api/src/main/java/com/velocitypowered/api/newcommand/BrigadierCommand.java

```diff
@@ -0,0 +1,45 @@
+package com.velocitypowered.api.newcommand;
+
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.suggestion.SuggestionProvider;
+import com.mojang.brigadier.suggestion.Suggestions;
+import com.mojang.brigadier.tree.LiteralCommandNode;
+import com.velocitypowered.api.command.CommandSource;
+
+/**
+ * A command that uses the Brigadier library for parsing.
```

Comment:
`A command that uses Brigadier for parsing the command and providing suggestions to the client.`